modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -0,0 +1,185 @@
+// Module included in the following assemblies:
+//
+// IPI
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+// * installing/ipi-install-post-installation-configuration.adoc
+// UPI
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * post_installation_configuration/post-install-bare-metal-configuration.adoc
+
+ifeval::["{context}" == "ipi-install-post-installation-configuration"]
+:postinstall-bare-metal-ipi:
+endif::[]
+ifeval::["{context}" == "post-install-bare-metal-configuration"]
+:postinstall-bare-metal-upi:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-customized-br-ex-bridge_{context}"]
+= Optional: Creating a manifest object that includes a customized `br-ex` bridge
+
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
```

Comment:
Just noticed this, and a lot of what's documented here is not postinstall activity. Is that going to cause a problem with the way this shows up in the docs?

Reply:
Ah, I missed that one is ifndef and the other is ifdef. Thanks!